cmds/webboot/webboot.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+func linkOpen(URL string) (io.ReadCloser, error) {
```

Comment:
I think it might be good to look at the URL package, it's pretty good.

https://play.golang.org/p/P-WuEwxoP41